comp/metadata/host/utils/host_info_windows.go

```diff
@@ -0,0 +1,115 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Portions of this code are taken from the gopsutil project
+// https://github.com/shirou/gopsutil .  This code is licensed under the New BSD License
+// copyright WAKAYAMA Shirou, and the gopsutil contributors
+
+package utils
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/gohai/platform"
+	"github.com/shirou/w32"
+	"golang.org/x/sys/windows"
+
+	"github.com/DataDog/datadog-agent/pkg/metadata/common"
+	"github.com/DataDog/datadog-agent/pkg/util/cache"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil"
+)
+
+var (
+	modkernel          = windows.NewLazyDLL("kernel32.dll")
+	procGetTickCount64 = modkernel.NewProc("GetTickCount64")
+)
+
+// InfoStat describes the host status.  This is not in the psutil but it useful.
+type InfoStat struct {
+	Hostname             string `json:"hostname"`
+	Uptime               uint64 `json:"uptime"`
+	BootTime             uint64 `json:"bootTime"`
+	Procs                uint64 `json:"procs"`           // number of processes
+	OS                   string `json:"os"`              // ex: freebsd, linux
+	Platform             string `json:"platform"`        // ex: ubuntu, linuxmint
+	PlatformFamily       string `json:"platformFamily"`  // ex: debian, rhel
+	PlatformVersion      string `json:"platformVersion"` // version of the complete OS
+	KernelVersion        string `json:"kernelVersion"`   // version of the OS kernel (if available)
+	KernelArch           string `json:"kernelArch"`
+	VirtualizationSystem string `json:"virtualizationSystem"`
+	VirtualizationRole   string `json:"virtualizationRole"` // guest or host
+	HostID               string `json:"hostid"`             // ex: uuid
+}
+
+// GetInformation just returns an InfoStat object, filled in with various
+// operating system metadata
+func GetInformation() *InfoStat {
+	info, _ := cache.Get[*InfoStat](
+		cacheKey,
+		func() (*InfoStat, error) {
+			info := &InfoStat{}
+			info.Hostname, _ = os.Hostname()
+
+			upTime := time.Duration(getTickCount64()) * time.Millisecond
+			bootTime := time.Now().Add(-upTime)
+			info.Uptime = uint64(upTime.Seconds())
+			info.BootTime = uint64(bootTime.Unix())
+			pids, _ := Pids()
+			info.Procs = uint64(len(pids))
+			info.OS = runtime.GOOS
+
+			info.KernelArch = runtime.GOARCH
+
+			pi := platform.CollectInfo()
+			info.Platform, _ = pi.OS.Value()
+			info.PlatformFamily, _ = pi.OS.Value()
```

Comment:
There is a `ValueOrDefault` function, and I guess using the `Family` field is more appropriate for the 2nd value
```suggestion
			info.Platform = pi.OS.ValueOrDefault()
			info.PlatformFamily = pi.Family.ValueOrDefault()
```

Reply:
Same as for the other comment, I don't want to introduce any behavior change in this PR.